conf/modules.config

```diff
@@ -1727,15 +1749,15 @@ process {
     }
 
     withName: GATK4_HAPLOTYPECALLER {
-        tag = { "${meta.reference}|${meta.sample_id}" }
+        tag = { params.genotyping_use_unmerged_libraries ? "${meta.reference}|${meta.sample_id}|${meta.library_id}" : "${meta.reference}|${meta.sample_id}" }
         ext.args = {[
             // Option names have changed from underscore_separated to hyphen-separated in GATK4
-            "--sample-ploidy ${meta2.ploidy}",
+            meta2.ploidy ? "--sample-ploidy ${meta2.ploidy}" : "--sample-ploidy 2",
```

Comment:
I think we should use the fasta schema to provide a default value for this instead of hard-coding default values everywhere.